### PR TITLE
NO_TICKET_REMOVE_GRPC: remove GRPC related code.

### PR DESCRIPTION
**Pull request by AbnerZheng**
Signed-off-by: Abner Zheng <abnerzheng@gmail.com>

### Overview
Remove all GRPC related code

### Which JIRA ticket does this PR relate to?
No ticket. 

### Complete this checklist before you submit this PR

- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [CasperLabs coding standards](https://casperlabs.atlassian.net/wiki/spaces/EN/pages/16842753/Coding+Standards).
- [ ] If this PR adds a new feature, it includes tests related to this feature.
- [ ] You assigned one person to review this PR.
- [ ] Your GitHub account is linked with our [Drone CI](https://drone-auto.casperlabs.io/) system. This is necessary to run tests on this PR.
- [ ] Do not forget to run `bors r+` if GitHub policy is not enforced, e.g. when merging into another feature branch. It may be omitted under some circumstances if this PR intentionally assumes that integration tests will fail but will be fixed with the future PRs.

### Notes

_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._

### CODE REVIEW

File: packages/sdk/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "casper-client-sdk",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (sacherjj) [inline packages/sdk/package.json]:
```suggestion
  "version": "1.0.2",
  "license": "Apache 2.0",
```
I misunderstood that the package was ready to release and bumped to 1.0.1 before finding the same grpc issue.  Need to bump to 1.0.2.  Also, need a license.

**Review by PfanP — APPROVED** [review]

File: packages/sdk/package.json (hunk 1 above)
Comment (AbnerZheng) [reply packages/sdk/package.json]: Thank you.

**Review by zie1ony — APPROVED** [review]
-1745 lines :+1:

**Pull request merged**